### ISSUE TITLE
Laggy interface with background on

### ISSUE THREAD

**Issue by HommeCrabe** [spocky/miproja1#125]
Hi! On my Fire TV Stick 4K Max (1st gen), the performance is not great. When I navigate in my categories, animations are not smooth, I'm probably at around 20-25 fps. Only way to fix this is to disable background. Then the app is smooth but feels a little empty :( I'm surprised a background, even the default one, could affect performance this much. Any idea how to improve this? Thank you for your support.

**Comment by Zizou3000** [comment]
I agree, and dynamic colors make it even worse. We really need a fix

**Comment by rr1231234** [comment]
Same issue. Glad I see this post.

**Comment by spocky** [comment]
Many improvements have been added recently. Does 4.62 improve smoothness ?

**Comment by HommeCrabe** [comment]
Indeed, latest version improves performance a lot. If someone else confirms, we could close this issue. Thank you for your work @spocky 🙂

**Comment by ethanbowering24** [comment]
I seem to be experiencing this same issue on v4.68, Onn 4k Pro with Android 12. Seting wallpaper to none dramatically improves the refresh rate.

**Comment by ethanbowering24** [comment]
After more fiddling it looks like setting "Shadow overlay opacity" to 0% also dramatically improves the refresh rate when using a wallpaper.
